examples/resources/genesyscloud_guide_version/locals.tf

```diff
@@ -0,0 +1,7 @@
+locals {
+  dependencies = {
+    resource = [
+      "../../data-sources/genesyscloud_guide/data-source.tf",
```

Comment:
Is the guide defined in the `data-source.tf` file referenced guaranteed to always be on the system? Otherwise you should probably use just a reference to the resource guide you have defined in the `resources` directory. You would also need to update the reference in this directory's `resource.tf` file as defined below